mail_notification_custom_subject/models/mail_message_custom_subject.py

```diff
@@ -31,8 +34,9 @@ class MailMessageCustomSubject(models.Model):
             ("append_before", "Append Before"),
             ("append_after", "Append After"),
             ("replace", "Replace"),
+            ("inside_replace", "Inside Replace"),
```

Comment:
suggestion: "Inside Replace" is hard to understand. Instead:

```suggestion
            ("replace", "Full Replacement"),
            ("inside_replace", "Partial Replacement"),
```